src/geometry/manifold/manifold.lean

```diff
@@ -203,13 +203,16 @@ instance : order_bot (structure_groupoid H) :=
   end,
   ..structure_groupoid.partial_order }
 
+instance (H : Type u) [topological_space H] : inhabited (structure_groupoid H) :=
+⟨id_groupoid H⟩
+
 /-- To construct a groupoid, one may consider classes of local homeos such that both the function
 and its inverse have some property. If this property is stable under composition,
 one gets a groupoid. `pregroupoid` bundles the properties needed for this construction, with the
 groupoid of smooth functions with smooth inverses as an application. -/
 structure pregroupoid (H : Type*) [topological_space H] :=
 (property : (H → H) → (set H) → Prop)
-(comp     : ∀{f g u v}, property f u → property g v → is_open (u ∩ f ⁻¹' v)
+(comp     : ∀{f g u v}, property f u → property g v → is_open u → is_open v → is_open (u ∩ f ⁻¹' v)
```

Comment:
I'm assuming this was a bug in the definition?

Reply:
This is a better definition like that, as it requires less and is enough for all the applications.